plugins/groovy/src/org/jetbrains/plugins/groovy/lang/completion/GrMethodOverrideCompletionProvider.java

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
I have fixed the other stuff you mentioned, but I honestly don't know what to test here as most of the functionality is delegated to already implemented and tested methods, and I don't know how to test what is left. I can upload what I already fixed.